editor/src/components/navigator/navigator-item/navigator-item-dnd-container.tsx

```diff
@@ -147,7 +140,7 @@ function onDrop(
 ): void {
   const dragSelections = propsOfDraggedItem.getCurrentlySelectedEntries()
   const filteredSelections = dragSelections.filter((selection) =>
-    notDescendant(propsOfDropTargetItem, selection.elementPath),
+    notDescendant(propsOfDropTargetItem.elementPath, selection.elementPath),
```

Comment:
this is used extensively now, it would make sense to extract it to a single function and reuse it

Reply:
e.g. line 115, 193, 292